src/server/editorServices.ts

```diff
@@ -536,15 +530,20 @@ module ts.server {
         updateProjectStructure() {
             this.log("updating project structure from ...", "Info");
             this.printProjects();
+            var openFilesReferenced: ScriptInfo[] = [];

```

Comment:
Consider calling this `updatedReferencedFiles` to disambiguate between it and `this.updatedReferencedFiles`. I actually thought we were mutating the original list at first.